myapp/src/App.js

```diff
@@ -0,0 +1,190 @@
+import logo from './logo.svg';
+import beverly from './Images/beverly.jpeg'
+import fall from './Images/fall.jpeg'
+import plants from './Images/plants.jpeg'
+import self from './Images/self.jpeg'
+import './App.css';
+
+function App() {
+
+ 
+  /*
+  //comment out the map and objects for now since they aren't necessary for the website
+  //this was just for testing out a map with objects
+  const maria = {
+    name: "maria",
+    age: 19,
+    major: "computer science",
+    location: "massachusetts"
+  }
+
+  const nonExistentSibling = {
+    name: "matthew",
+    age: 22,
+    major: "biology",
+    location: "new york"
+
+  }
+
+  const kevin = {
+    name: "kevin",
+    age: 19,
+    major: "mechanical engineering",
+    location: "massachusetts"
+  }
+
+  const sarah = {
+    name: "sarah",
+    age: 19,
+    major: "english",
+    location: "delaware"
+  }
+
+  //create a map with objects
+  const friends_and_family = [nonExistentSibling, kevin, sarah];
+  const friendMap = friends_and_family.map((friend) =>{
+   
+    return (
+      
+      <h1>{friend.name}</h1>
+    )
+  });
+
+  */
+
+  //create a map
+  const foodsList = ["Apple Pie", "Pasta Salad", "Omelettes" ];
+  const foodMap = foodsList.map((food) =>{
+    return(
+      <p className="foodMapText">{food}</p>
+    )
+  });
+
+
+  //function to do something when button is clicked, have different methods for each image
+  const bevImgFunction = () => {
+    var x = document.getElementById("bevButton");
+
```

Comment:
love it! this is a really creative way to use buttons where we are rn in the course!

also great job using arrow functions